Sprint-1/interpret/to-pounds.js

```diff
@@ -15,7 +15,7 @@ const pence = paddedPenceNumberString
   .substring(paddedPenceNumberString.length - 2)
   .padEnd(2, "0");
```

Comment:
Just an extra question. Do you think .padEnd(2, "0") is necessary?

Reply:
it is not necessary but when its only one character in the pence side it will add a "0"
for example if the pence side is only a "6" pence it makes it "60" pence

Reply:
Have you tried changing the value of `penceString` to check whether your expectation holds ( without `.padEnd(2, "0")` )?

Reply:
Yes I changed it to different numbers and the answer were correct